packages/lib/src/vis/rgb/utils.ts

```diff
@@ -2,7 +2,7 @@ import type { NumArray } from '@h5web/shared';
 import { getDims, toTypedNdArray } from '@h5web/shared';
 import type { NdArray } from 'ndarray';
 import ndarray from 'ndarray';
-import { DataTexture3D, FloatType, RedFormat, UnsignedByteType } from 'three';
+import { Data3DTexture, FloatType, RedFormat, UnsignedByteType } from 'three';
```

Comment:
`DataTexture3D` was renamed to `Data3DTexture` in r138: https://github.com/mrdoob/three.js/wiki/Migration-Guide#137--138

I renamed our function to match it